testsuite/tests/kuadrant/authorino/operator/http/conftest.py

```diff
@@ -0,0 +1,38 @@
+"""Conftest for all tests requiring custom deployment of Authorino"""
+import pytest
+from weakget import weakget
+
+from testsuite.openshift.objects.authorino import AuthorinoCR
+from testsuite.objects import Authorino, Authorization
+from testsuite.openshift.objects.auth_config import AuthConfig
+from testsuite.httpx import HttpxBackoffClient
+
+# pylint: disable=unused-argument
+@pytest.fixture(scope="module")
+def authorization(wildcard_domain, blame, openshift, rhsso_service_info, module_label) -> Authorization:
+    """In case of Authorino, AuthConfig used for authorization"""
+    authorization = AuthConfig.create_instance(openshift, blame("ac"), wildcard_domain, labels={"testRun": module_label})
```

Comment:
There is a different solution that doesn´t require creating new `AuthConfig` and using existing one. (I came up with that only after we talked, sorry)

```
authorization.remove_all_hosts()
authorization.add_host(wildcard_domain)
```